modern/src/reports/ReportFilter.js

```diff
@@ -4,7 +4,7 @@ import t from '../common/localization';
 import { useSelector } from 'react-redux';
 import moment from 'moment';
 
-const ReportFilter = ({ children, handleSubmit, showOnly }) => {
+const ReportFilter = ({ children, handleSubmit, showOnly, showDevices }) => {
```

Comment:
If you want to re-use the filter, I would recommend adding more nesting. I don't like adding another configuration here.

For example, you can have `ReportFilter` without device and `DeviceReportFilter` with device that uses `ReportFilter`.

Reply:
I would not like to complicate things for now. Just separated the statistics page and removed the configuration from ReportFilter.

![image](https://user-images.githubusercontent.com/41992346/108339713-c6bef500-71fd-11eb-976a-1fdb462f7033.png)

Reply:
Can we revert these changes then?

Reply:
Done.

Reply:
Doesn't looks like it's reverted.